### PR TITLE
New package: passff-host-1.2.1

### PR DESCRIPTION
**Pull request by vpzomtrrfrt**

### CODE REVIEW

File: srcpkgs/passff-host/template [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Template file for 'passff-host'
+pkgname=passff-host
+version=1.2.1
+revision=1
+hostmakedepends="make"
+makedepends=""
```
Comment (pullmoll) [inline srcpkgs/passff-host/template]: Remove this line.

File: srcpkgs/passff-host/template [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Template file for 'passff-host'
+pkgname=passff-host
+version=1.2.1
+revision=1
+hostmakedepends="make"
```
Comment (pullmoll) [inline srcpkgs/passff-host/template]: The package `make` is already in `base-chroot` so you can remove this line.

File: srcpkgs/passff-host/template [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+makedepends=""
+depends="python3"
+short_desc="Host app for the WebExtension PassFF"
+maintainer="Colin Reeder <colin@vpzom.click>"
+license="GPL-2.0"
+homepage="https://github.com/passff/passff-host"
+distfiles="https://github.com/passff/passff-host/releases/download/$version/passff.json https://github.com/passff/passff-host/releases/download/$version/passff.py"
+checksum="cd871bcf29d71e53f986fb74b7f2b1867151ba2b2a27318044c7bfaa38827282 6db5c9c435bfe8efc0f626e38144eff4c045d90d8508a295c990abbae411ac6c"
+skip_extraction="passff.json passff.py"
+create_wrksrc=yes
```
Comment (pullmoll) [inline srcpkgs/passff-host/template]: Move this line up to before `depends=`. You should install `xtools` and run `xlint srcpkgs/passff-host/template` to see what and how to fix in your template.

**Comment by pullmoll** [comment]
Please squash the commits.

**Comment by xtraeme** [comment]
What @pullmoll said

**Comment by vpzomtrrfrt** [comment]
Squashed

File: srcpkgs/passff-host/template [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# Template file for 'passff-host'
+pkgname=passff-host
+version=1.2.1
+revision=1
+create_wrksrc=yes
+depends="python3"
+short_desc="Host app for the WebExtension PassFF"
+maintainer="Colin Reeder <colin@vpzom.click>"
+license="GPL-2.0-or-later"
```
Comment (Chocimier) [inline srcpkgs/passff-host/template]: -only

File: srcpkgs/passff-host/template [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# Template file for 'passff-host'
+pkgname=passff-host
+version=1.2.1
+revision=1
+create_wrksrc=yes
+depends="python3"
```
Comment (Chocimier) [inline srcpkgs/passff-host/template]: pass is needed too

File: srcpkgs/passff-host/template [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+short_desc="Host app for the WebExtension PassFF"
+maintainer="Colin Reeder <colin@vpzom.click>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/passff/passff-host"
+distfiles="https://github.com/passff/passff-host/releases/download/$version/passff.json https://github.com/passff/passff-host/releases/download/$version/passff.py"
```
Comment (Chocimier) [inline srcpkgs/passff-host/template]: distfiles and checksum should have one entry per line, indented with one space

File: srcpkgs/passff-host/template (hunk 4 above)
Comment (vpzomtrrfrt) [reply srcpkgs/passff-host/template]: I wasn't sure which one to choose, how is that determined?

Comment (Chocimier) [reply srcpkgs/passff-host/template]: Citing GPL, 

> Each version is given a distinguishing version number.  If the Program
specifies a version number of this License which applies to it and "any
later version", you have the option of following the terms and conditions
either of that version or of any later version published by the Free
Software Foundation.  If the Program does not specify a version number of
this License, you may choose any version ever published by the Free Software
Foundation.

**Pull request merged**